### DISCUSSION TITLE
Custom Hdls and Ping Srvice

### DISCUSSION THREAD

**Discussion by jairogutierrezpayu** [arkmq-org/arkmq-org-broker-operator#434] [Q&A] [answered]
Hi, as part of operator i have two services created _xxx-hdks-svc_ and _xxx-ping-svc_, this services are ClusterIP type. How i can change this services for set LoadBalancer type

Thx

**Comment by gaohoward** [comment, marked as answer]
Hi if you mean the services created by the operator, they are not supposed to be used as loadbalancer. I think you need to apply your own loadbalancer.

**Reply by Zodor** [reply]
Hi. I also want to change the ClusterIP typ to LoadBalancer (or at least NodePort), becuase AWS Loadbalancer won't accept that the service-type is ClusterIP. I have tried to change it myself, and it works like a charm (for some time). Then I think that the operator is changing it back to the original ClusterIP.

I have not idea how to make the port 61616 available from outside my cluster if I cannot accomplish this. We use the external-dns pods to automatically create route53 dns records and automatically create the LoadBalancer when the brokers go online. That can only be accomplished if we have the opportunity to change whatever annotation or spec that we want on the services for Artemis
